domain/model/bootstrap/bootstrap.go

```diff
@@ -127,7 +128,7 @@ func CreateReadOnlyModel(
 		}
 
 		var m coremodel.Model
-		err := controllerDB.StdTxn(ctx, func(ctx context.Context, tx *sql.Tx) error {
+		err := controllerDB.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
```

Comment:
Thanks for this.

Reply:
Theres a lot still to go in this domain :)))